.changeset/forty-ligers-dance.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/typography': minor
+---
+
+Updates disabled Label & Description color in dark mode from `gray.base` to `gray.dark1`
```

Comment:
Did we have a ticket for this? Just wondering because seeing this makes me nervous about whether this will break styles in other components.

Reply:
That's a good catch. I assumed this was a global change, but it's just for Checkbox. Will revert